rules/windows/credential_access_posh_request_ticket.toml

```diff
@@ -14,6 +14,41 @@ index = ["winlogbeat-*", "logs-windows.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "PowerShell Kerberos Ticket Request"
+note = """## Triage and analysis
+
+### Investigating Explicit PowerShell Kerberos Ticket Request
+
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks, making
+it available for use in various environments, creating an attractive way for attackers to execute code.
+
+When an account has an SPN associated, it makes a viable target for a Kerberoasting attack, which consists of brute
+force to crack the user password, which is used to encrypt a Kerberos TGS ticket.
+
+Attackers can use PowerShell to request these Kerberos tickets, with the intent of extracting them from memory to
+perform Kerberoasting.
+
+#### Possible investigation steps:
+
+- Retrieve the script contents.
+- Investigate script execution chain (parent process tree).
+- Investigate if the script was executed and identify which account is targeted.
+- Check whether this user should be doing this kind of activity.
+- Contact the account owner and confirm whether they are aware of this activity.
+- Check if the script has any other functionality that can be potentially malicious.
+- Investigate other alerts related to the host and user in the last 48 hours.
+- Review event ID 4769 related to this account and service name for additional information
+### False Positive Analysis
+
+- A possible false positive can be identified if the script content is not malicious/harmful or does not request
+Kerberos tickets for user accounts, as computer accounts are not vulnerable to Kerberoasting due to complex password
```

Comment:
```suggestion
Kerberos tickets for user accounts, as host-based accounts are not vulnerable to Kerberoasting due to complex password
```
I could be wrong about this terminology. I got it from https://www.qomplx.com/qomplx-knowledge-kerberoasting-attacks-explained/

Reply:
https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/service-accounts-computer